Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -1212,6 +1212,7 @@ - (void)viewDidAppear:(BOOL)animated {
     }
     [self updateNavigationBarSubtitleLabel];
     [MarkIdentityAsSeenJob runWithThread:self.thread];
+    [ProfileFetcherJob runWithThread:self.thread networkManager:self.networkManager];
```

Comment:
Not objecting, but is "viewDidAppear" the right place?  I think it means we'll re-fetch whenever we close conversation settings or the app returns from the background in this conversation.  The latter seems right but maybe not the former?  But the former should be rare so... fine.

Oh, the former covers the "...When we add a member to a group?" case.  Makes sense.